test/webRDFS/expectedResultCreate

```diff
@@ -0,0 +1,4 @@
+HTTP/1.1 200 OK
+Content-Length: 13
+
+/fileToCreate
```

Comment:
This doesn't seem to conform to the [webHDFS API for create](https://hadoop.apache.org/docs/r1.0.4/webhdfs.html#CREATE).

It should return HTTP 201 with a zero content length and an empty response body.

Reply:
Should it also include the location?

Reply:
I don't think so; the API spec suggests that the response body is always empty